pkg/buffers/buffer/buffer.go

```diff
@@ -29,7 +29,7 @@ func (b *Buffer) RecordMetric() {
 	dur := time.Since(b.checkedOutAt)
 	checkoutDuration.Observe(float64(dur.Microseconds()))
 	checkoutDurationTotal.Add(float64(dur.Microseconds()))
-	totalBufferSize.Add(float64(b.Len()))
+	totalBufferSize.Add(float64(b.Cap()))
```

Comment:
you love sneaking in these lil' bug fixes

Reply:
Ahh it's because the base PR actually reverted this change. I'll update the base and this should get fixed. Thanks for catching it.